openml_OS/views/pages/api_new/v1/xml/pre.php

```diff
@@ -292,6 +293,8 @@
 $this->apiErrors[544] = 'Illegal filter specified';
 $this->apiErrors[545] = 'Can not specify offset without limit';
 $this->apiErrors[546] = 'Requested result limit too high. ';
+$this->apiErrors[547] = 'Per fold can only be set to value "true" or "false". ';
+$this->apiErrors[548] = 'Per fold queries are experimental and require a fair amount of filters on resulting run records to keep the query fast (use, e.g., flow, setup, task and uploader filter)';
```

Comment:
unclear error message. You want to say 'you cannot use filter X without also using filter Y,Z'?